dts/bindings/memory-window/intel,adsp-mem-window.yaml

```diff
@@ -0,0 +1,25 @@
+description: ADSP Memory Window
+
+compatible: "intel,adsp-mem-window"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  initialize:
+    type: boolean
+    description: Initialize memory region and flush cache.
+
+  memory:
+    required: true
+    type: phandle
+    description: memory region the windows are exposed through.
+
+  offset:
+    type: int
+    description: offset from memory base.
```

Comment:
what's the intent of offset, why can't `memory` just start at offset?  Seems like mixing of describing of the memory region, vs intent of some section of the memory region.

Reply:
memory is defined in some other node, offset is used here as an override and to deal with different layouts on different plaforms, the offset is actually calculated based on Kconfig, because it depend on what the application wants to do.